src/rp-paradigm/src/index.js

```diff
@@ -0,0 +1,62 @@
+const rxjs = require('rxjs');
+const { fromEvent, BehaviorSubject } = rxjs;
+
+const generateTodoList = (todos) =>
+    todos.map(({ date, value }, i) => {
+        const todo = document.createElement('div');
+        todo.className = 'todo-item';
+
+        const todoDate = generateSpanNode(date, 'todo-item__create-date');
+        const todoTask = generateSpanNode(value, 'todo-item__task');
+
+        todo.append(todoDate, todoTask);
+
+        return todo;
+    });
+
+const rerender = (todos) => {
+    const todoList = document.querySelector('.todo-list');
+    todoList.innerHTML = '';
+
+    todoList.append(...generateTodoList(todos));
+};
+
+const todosSubject = new BehaviorSubject([]);
+let todos = [];
+
+todosSubject.subscribe((_todos) => {
+    console.log(_todos);
+    todos = _todos;
+    rerender(todos);
+});
```

Comment:
Here am using, BehaviourSubject and after subscription, we change data of `todos`. After each change, we call `rerender` function, which rerenders all DOM elements of `.todo-list` class.